engine/script/src/script_timer.cpp

```diff
@@ -296,7 +296,7 @@ namespace dmScript
         return timer->m_Handle;
     }
 
-    bool CancelTimer(HTimerWorld timer_world, HTimer handle)
+    bool CancelTimer(HTimerWorld timer_world, HTimer handle, bool finish)
```

Comment:
I think "finish" is not the best name, as it's _not_ finished (it is in fact cancelled, as it says in the function name).

What the bool does though, is control whether the callback is called.
So perhaps something along the lines of "do_callback" or "send_callback"?